napi-inl.h

```diff
@@ -425,6 +425,38 @@ inline std::u16string String::Utf16Value() const {
 // Object class
 ////////////////////////////////////////////////////////////////////////////////
 
+namespace helper {
```

Comment:
Though I'm not sure this class really needs to be "hidden" in a namespace like this, since it is sort of part of the public API.